src/web/app/page.component.html

```diff
@@ -42,25 +42,67 @@
   </div>
 </nav>
 <div id="main-content" class="container main-content">
-  <div *ngIf="isUnsupportedBrowser">
-    You are currently using {{ browser }}. This web browser is not officially supported by TEAMMATES.
-    In case this web browser does not display the webpage correctly you may wish to view it in the following supported browsers:
-    <ol>
-      <li>Microsoft Internet Explorer {{ minimumVersions['IE'] }}+</li>
-      <li>Chrome {{ minimumVersions['Chrome'] }}+</li>
-      <li>Firefox {{ minimumVersions['Firefox'] }}+</li>
-      <li>Safari {{ minimumVersions['Safari'] }}+</li>
-    </ol>
-    <br>
+  <div *ngIf="isUnsupportedBrowser && !alertDismissed">
+    <ngb-alert type="info" (close)="closeAlert()">
```

Comment:
Not sure if this is the best way; there is nothing special about this particular display message that it can be dismissed, but not others. Let's remove the removable feature for now.

Reply:
Okay, removed.

I initially thought that people who use old browsers by default may be hindered by the constant display of this message (but this is a rare case plus yeah this are better ways to do this)